pages/common/kdeconnect-cli.md

```diff
@@ -0,0 +1,35 @@
+# kdeconnect-cli
+
+> KDE Connect CLI.
```

Comment:
Let's use that manned.org link without the commit hash. Then it'll stay up to date?

Reply:
It's still from 2014 and only has only 6 of 24 flags documented. I wouldn't call that "More" information.

Reply:
So I think the best is the official website:
```suggestion
> KDE Connect CLI.
> More information: <https://kdeconnect.kde.org/>.
```

Reply:
Probably the best option we have.

Reply:
Don't they just use man pages from other sources?

Reply:
No, just the official of the command, and the command doesn't have a man page. :man_shrugging: thread closed :joy: